.devops/code-review-pipelines.yml

```diff
@@ -85,4 +85,10 @@ stages:
               API_TEST_FILE: "api-tests/wallet_local_paypal.collection.tests.json"
               TEST_FILE_PREFIX: "wallet-api-test-paypal"
               WALLET_TOKEN_TEST: $(WALLET_TOKEN_TEST_DEV)
+          - template: azure-templates/api-tests.yml
```

Comment:
You could add a **TEST_DISPAY_NAME** variable to the **api-tests.yml** template to change the **displayName** used to differentiate the execution of the various tests. Currently in the **[CR](https://dev.azure.com/pagopaspa/pagoPA-projects/_build/results?buildId=145054&view=logs&j=5a5405a0-8439-550a-2a82-0e4c8988d6a1&t=8a1a1b8e-2084-583d-18a8-37c9f1732422)** pipeline they are very confusing, in fact, it always shows the displayName 'Run api test' 🙂